modules/standard/Math.chpl

```diff
@@ -1056,6 +1056,25 @@ module Math {
     extern proc truncf(x: real(32)): real(32);
     return truncf(x);
   }
+  
+  /* Returns the greatest common divisor of the integer argument `arg1` and
+     `arg2`. */
+  proc gcd(in a: int,in b: int): int {
+     if(a < 0) {
+     a = -a;
+     }
+     if(b < 0) {
+     b = -b;
+     }
+     var r: int;
+     while(b != 0) {
+       r = a % b;
+       a = b;
+       b = r;
+     }
+    return abs(a);
```

Comment:
Since everything up to this point is positive, we don't need this final abs, right?

Reply:
Oh.. sorry sir !! I will update this thanks !!